riak_test/yz_faceted_search.erl

```diff
@@ -120,15 +120,24 @@ verify_field_faceting(Cluster, Index) ->
                             <<"facet_fields">>,
                             <<"state">>],
                            Struct),
-    ?assertEqual([<<"Ohio">>,4,<<"Kentucky">>,1], StateCounts).
+    %% We expect to see all 4 Ohio restaurants in the search results, but none
+    %% of the Kentucky restaurants because there is only one in the test data
+    %% set, and facet.mincount is set to 3.
+    ?assertEqual([<<"Ohio">>,4], StateCounts).
 
 -define(PRICE_RANGE_1, <<"price:[1 TO 29]">>).
 -define(PRICE_RANGE_2, <<"price:[30 TO 100]">>).
 verify_query_faceting(Cluster, Index) ->
     HP = yz_rt:host_port(Cluster),
     Params = [{facet, true},
+              {'facet.mincount', 1},
               {'facet.query', "price:[1 TO 29]"},
               {'facet.query', "price:[30 TO 100]"}],
+    %% Verify 10 times because of non-determinism in coverage
+    [verify_query_facets(HP, Index, Params) || _ <- lists:seq(1, 10)],
```

Comment:
I realize we use this pattern elsewhere, but I wonder if we should be checking the response, which I believe has the query plan that was used, and wait for the plan to change.  Not a blocker by any means, but we should consider a strategy for doing this test more deterministically in the future.